### ISSUE TITLE
Alternative text for buttons

### ISSUE THREAD

**Issue by jokd** [origo-map/origo#1006]
**Is your feature request related to a problem? Please describe.**
To comply with WCAG 2.0, sc 4.1.2 the map should have alternative text for the buttons. As now there are just tooltips that aren´t compatible with screen readers (I have been told).

**Describe the solution you'd like**
Put alternative texts on buttons and other elements.
